packages/platform/src/lib/form-group.ts

```diff
@@ -195,6 +196,15 @@ export function createFormGroup<FormFields extends FormGroupCreator>(
       }
       Object.values(fg).forEach((f) => markFormControlAsTouched(f));
     },
+    markAsPristine: () => {
+      const fg = isSignal(formFieldsMapOrSignal)
+        ? formFieldsMapOrSignal()
+        : formFieldsMapOrSignal;
+
+      Object.values(fg).forEach((f) => {
+        f.markAsPristine();
+      });
```

Comment:
`fg` can be an array, in this case we don't need to invoke `Object.values`.
This can be rewritten as follows (similar to `markAllAsTouched`).
```suggestion
      if (Array.isArray(fg)) {
        fg.forEach((f) => markFormControlAsTouched(f));
        return;
      }
      Object.values(fg).forEach((f) => markFormControlAsTouched(f));
```